packages/@ourworldindata/grapher/src/captionedChart/CaptionedChart.tsx

```diff
@@ -469,12 +539,25 @@ export class StaticCaptionedChart extends CaptionedChart {
                 />
                 {this.header.renderStatic(paddedBounds.x, paddedBounds.y)}
                 {this.renderChart()}
-                {this.footer.renderStatic(
-                    paddedBounds.x,
-                    paddedBounds.bottom - this.footer.height
-                )}
-                {this.renderSVGDetails()}
+                <StaticFooter
+                    manager={manager}
+                    maxWidth={maxWidth}
+                    targetX={paddedBounds.x}
+                    targetY={paddedBounds.bottom - this.staticFooter.height}
+                />
+                {includeDetailsInStaticExport && this.renderSVGDetails()}
             </svg>
         )
     }
 }
+
+function VerticalSpace({ height }: { height: number }): JSX.Element {
```

Comment:
Note for the reviewers: Although a bit unconventional, adding vertical space as a `<div />` makes margin collapsing impossible and makes it easier to track the space available for the chart ares (see `chartHeight` function)

Reply:
Good point!